wrapper.go

```diff
@@ -76,10 +74,6 @@ func New(url string) (ws *WebSocket, err error) {
 type WebSocket struct {
 	*js.Object
 
-	// Available events:
-	// open, error, close, message
```

Comment:
Agreed, in https://github.com/gopherjs/websocket/pull/15/commits/bc008c5686cb853bf8266921c04d944330ccb3b8 I tried to create sensible function documentation for ```AddEventListener``` and ```RemoveEventListener``` which preserves this information directly in source (although it is also available for users in doc.go).